qutebrowser/mainwindow/tabwidget.py

```diff
@@ -517,6 +513,7 @@ def tabSizeHint(self, index):
 
             no_pinned_count = self.count() - self.pinned_count
             pinned_width = tab_width_pinned_conf * self.pinned_count
+            # Prevent any tabs from being smaller than the min size
             no_pinned_width = self.width() - pinned_width
 
             if pinned:
```

Comment:
For this case here, we set the width to whatever the user gives us, but should we be calculating the proper width (accouting for small windows)? Right now, if we overallocate, the tab bar seems to shrink properly, is that OK behavior?